test/upgrade/2.0.0-2.1.0.js

```diff
@@ -0,0 +1,1108 @@
+      bundlesState.push(await bundleHandler.connect(rando).getBundle(id));
+      bundleIdByOfferState.push(await bundleHandler.connect(rando).getBundleIdByOffer(id));
+      bundleIdByTwinState.push(await bundleHandler.connect(rando).getBundleIdByTwin(id));
+    }
+
+    let nextBundleId = await bundleHandler.connect(rando).getNextBundleId();
+    return { bundlesState, bundleIdByOfferState, bundleIdByTwinState, nextBundleId };
+  }
+
+  async function getConfigContractState() {
+    return {
+      tokenAddress: await configHandler.connect(rando).getTokenAddress(),
+      treasuryAddress: await configHandler.connect(rando).getTreasuryAddress(),
+      voucherBeaconAddress: await configHandler.connect(rando).getVoucherBeaconAddress(),
+      beaconProxyAddress: await configHandler.connect(rando).getBeaconProxyAddress(),
+      protocolFeePercentage: await configHandler.connect(rando).getProtocolFeePercentage(),
+      protocolFeeFlatBoson: await configHandler.connect(rando).getProtocolFeeFlatBoson(),
+      maxOffersPerBatch: await configHandler.connect(rando).getMaxOffersPerBatch(),
+      maxOffersPerGroup: await configHandler.connect(rando).getMaxOffersPerGroup(),
+      maxTwinsPerBundle: await configHandler.connect(rando).getMaxTwinsPerBundle(),
+      maxOffersPerBundle: await configHandler.connect(rando).getMaxOffersPerBundle(),
+      maxTokensPerWithdrawal: await configHandler.connect(rando).getMaxTokensPerWithdrawal(),
+      maxFeesPerDisputeResolver: await configHandler.connect(rando).getMaxFeesPerDisputeResolver(),
+      maxEscalationResponsePeriod: await configHandler.connect(rando).getMaxEscalationResponsePeriod(),
+      maxDisputesPerBatch: await configHandler.connect(rando).getMaxDisputesPerBatch(),
+      maxTotalOfferFeePercentage: await configHandler.connect(rando).getMaxTotalOfferFeePercentage(),
+      maxAllowedSellers: await configHandler.connect(rando).getMaxAllowedSellers(),
+      buyerEscalationDepositPercentage: await configHandler.connect(rando).getBuyerEscalationDepositPercentage(),
+      authTokenContractNone: await configHandler.connect(rando).getAuthTokenContract(AuthTokenType.None),
+      authTokenContractCustom: await configHandler.connect(rando).getAuthTokenContract(AuthTokenType.Custom),
+      authTokenContractLens: await configHandler.connect(rando).getAuthTokenContract(AuthTokenType.Lens),
+      authTokenContractENS: await configHandler.connect(rando).getAuthTokenContract(AuthTokenType.ENS),
+      maxExchangesPerBatch: await configHandler.connect(rando).getMaxExchangesPerBatch(),
+      maxRoyaltyPecentage: await configHandler.connect(rando).getMaxRoyaltyPecentage(),
+      maxResolutionPeriod: await configHandler.connect(rando).getMaxResolutionPeriod(),
+      minDisputePeriod: await configHandler.connect(rando).getMinDisputePeriod(),
+      accessControllerAddress: await configHandler.connect(rando).getAccessControllerAddress(),
+    };
```

Comment:
maybe this is another case we can take advantage of Promise.all

Reply:
Here promise.all at least theoretically makes sense.
I changed it (also in some other getters).

But gain was only ~ 1s, which is negligible compared to other setup time (~40s). Anyway, since I made the changes, and I'll be using `getProtocolContractState` in other tests, I'm keeping the chnage.